tensor_test.go

```diff
@@ -10,3 +10,17 @@ func ExampleTensor() {
 	t.Close()
 	// Output:
 }
+
+func ExampleTranspose2d() {
+	input := torch.RandN([]int{1, 1, 1}, false)
+	weight := torch.RandN([]int{1, 3, 3}, false)
+	var bias torch.Tensor
+	stride := []int{1}
+	padding := []int{0}
+	outputPadding := []int{0}
+	groups := 1
+	dilation := []int{1}
+	out := torch.ConvTranspose2d(input, weight, bias,
+		stride, padding, outputPadding, groups, dilation)
+	out.Print()
```

Comment:
printing the Tensor has no help to check the function work well, do we have another way to check it?

Reply:
As we can't specify values to tensor now, we can't validate the output. So maybe, we can only test if there is no error occurs when running test.